src/node_sea.cc

```diff
@@ -414,7 +414,9 @@ std::optional<std::string> GenerateCodeCache(std::string_view main_path,
   RAIIIsolate raii_isolate(SnapshotBuilder::GetEmbeddedSnapshotData());
   Isolate* isolate = raii_isolate.get();
 
+  v8::Isolate::Scope isolate_scope(isolate);
```

Comment:
What do you think about moving the `isolate_scope` into the `RAIIIsolate` class? Since `RAIIIsolate` is just a utility class which helps in creating an Isolate that can be used instantly, it seems like if we add it there, it would be beneficial.

Reply:
Sure. Note that you already need to allocate a `HandleScope` to do anything interesting with `RAIIIsolate`. Would you propose pushing that into the ctor of RAIIIsolate too?